persistence-cassandra/core/src/main/resources/play/reference-overrides.conf

```diff
@@ -1,6 +1,6 @@
 
 # Make the stash bounded in size 
-akka.actor.default-mailbox.stash-capacity=10000
+akka.actor.default-mailbox.stash-capacity = 10000
```

Comment:
Unintentional format? 
Mental note: do we have a formatting process for HOCON files?